src/products/ussd/expressHandler.ts

```diff
@@ -0,0 +1,31 @@
+import joi from 'joi';
```

Comment:
Maybe we should deprecate this? Not sure if it's used much ... just a thought for now